src/main/webapp/app/entities/feedback-suggestion.model.ts

```diff
@@ -0,0 +1,30 @@
+import { GradingInstruction } from 'app/exercises/shared/structured-grading-criterion/grading-instruction.model';
+
+export class TextFeedbackSuggestion {
+    constructor(
+        public id: number | undefined,
+        public exerciseId: number,
+        public submissionId: number,
+        public title: string,
+        public description: string,
+        public credits: number,
+        public gradingInstruction: GradingInstruction | undefined,
+        public indexStart: number | undefined,
+        public indexEnd: number | undefined,
+    ) {}
+}
+
+export class ProgrammingFeedbackSuggestion {
+    constructor(
```

Comment:
Note for reviewers: This is currently pretty much unused. It will make sense in combination with #7099